test/Ocelot.AcceptanceTests/MapRequestTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.AcceptanceTests
```

Comment:
File-scoped declaration should be!

Reply:
Fixed in https://github.com/alexreinert/Ocelot/commit/53e49d1ca701e8185e5f26a460f3892406470064